doc/how_nml_works.md

```diff
@@ -32,3 +32,8 @@ In order to make a client using NeosModLoader compatible with vanilla Neos the c
 [components]: https://wiki.neos.com/Component
 [logix]: https://wiki.neos.com/LogiX
 [data model]: https://wiki.neos.com/Core_Concepts#Data_Model
+
+## Other Minor Ways NeosModLoader Touches Neos
```

Comment:
Reads weird, may want to reuse 'Interfaces with' or list it as addition hooks.
Alternatively could refer to the main ones as 'Primary' and these as 'Secondary' hooks. 
Primary as required to function
Secondary as QoL/additional features

Reply:
Fixed in  3c4e40d0